### PR TITLE
Please set getResponseStatusCode as public

### PR DESCRIPTION
**Pull request by FanchTheSystem**
Because I have in some of my Helper the same code as in this method witch I can't call as it is protected

### CODE REVIEW

**Comment by FanchTheSystem** [comment]
Example method I already have in my helper:  

``` php
public function getStatusCode()
    {
        $response = $this->getModule('PhpBrowser')->client->getInternalResponse();
        if (method_exists($response, 'getStatus')) {
            return $response->getStatus();
        }
        if (method_exists($response, 'getStatusCode')) {
            return $response->getStatusCode();
        }
        return null;
    }
```


Example method I would like to have in my helper:
``` php
public function getStatusCode()
    {
        return = $this->getModule('PhpBrowser')->getResponseStatusCode();
    }
```

**Comment by Naktibalda** [comment]
Internal response of PhpBrowser always returns an object with getStatus method.
You only need to use getStatusCode with Symfony module (probably Laravel, Silex and Lumen too).

https://github.com/symfony/browser-kit/blob/4.0/Response.php#L87
https://github.com/symfony/http-foundation/blob/4.0/Response.php#L475

File: src/Codeception/Lib/InnerBrowser.php [hunk 1]
```diff
@@ -1244,7 +1244,7 @@ protected function debugResponse($url)
         $this->debugSection('Response Headers', $this->getRunningClient()->getInternalResponse()->getHeaders());
     }
 
-    protected function getResponseStatusCode()
+    public function getResponseStatusCode()
```
Comment (DavertMik) [inline src/Codeception/Lib/InnerBrowser.php]: In this case method name should be prefixed with `_` so it won't go to a tester class and would be accessible only from helpers.

Comment (FanchTheSystem) [reply src/Codeception/Lib/InnerBrowser.php]: Like https://github.com/Codeception/Codeception/pull/4745/commits/8f4a04970dbcd46bd1a8d99d4197deea57b5302a ?

**Comment by DavertMik** [comment]
Thanks!

**Pull request merged**